api/src/main/java/org/openmrs/module/fhir/api/util/ErrorUtil.java

```diff
@@ -0,0 +1,18 @@
+package org.openmrs.module.fhir.api.util;
+
+import java.util.List;
+
+public class ErrorUtil {
```

Comment:
This class can be final and should have private constructor.

Reply:
Ok